CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+1.4.0 / 2019-08-28
+==================
+
+Bug fixes:
+* Don't use TABs, when triggering `change` JS event upon input value is change (fixes auto-complete control testing in Google Chrome)
+* Fixed inability to manipulate windows when Selenium 3 with Firefox GeckoDriver was used
+* The `clickOnElement` method wasn't working when Selenium 3 with Firefox GeckoDriver was used
+
+Testsuite:
+* Don't test on PHP 5.3 (driver itself would likely continue to work on PHP 5.3 for some time)
+* Adding testing on PHP 7.1, 7.2, 7.3 and 7.4beta
+* PhantomJS was removed from testsuite.
+
+Changes:
+* Bumped requirement to PHP 5.4
+* Minors improvements to driver when running against W3C compatible driver
+
+New features:
+* Allow uploading files to remote Selenium instances (e.g. SauceLabs, BrowserStack, etc.)
+* Added `getDesiredCapabilities` method for fetching current desired capabilities
+
+Misc:
+* Syn library is [0.0.3](https://github.com/bitovi/syn/tree/v0.0.3)
+* The `setDesiredCapabilities` method now combines default capabilities with default ones
```

Comment:
Please replace `combines default capabilities with default ones` with `combines default capabilities with user provided ones`.

---

I was probably sleeping while writing these, because it sounds very odd. Sorry.

Reply:
updated